src/gemm_v2.cpp

```diff
@@ -429,7 +429,7 @@ miopenStatus_t CallGemm(const Handle& handle,
                 rocblas_datatype::rocblas_datatype_i32_r,
                 rocblas_gemm_algo::rocblas_gemm_algo_standard,
                 0,
-                0);
+                rocblas_gemm_flags_pack_int8x4);
```

Comment:
Add something like
```
#define USE_GEMM_FLAGS_PACK_INT8X4 (MIOPEN_ROCBLAS_VERSION_DECIMAL >= 238)
```
and use it to maintain backward compatibility with previous rocBLAS. Their changelog says this begins from 2.38, https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/72a59738b291e642f5fecb0b2d5ab2123a2e7793/CHANGELOG.md.

Example:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/bdd81a25f0dd6a913c2c2636ecd2c3edd1baf733/src/gemm_v2.cpp#L57-L59

Reply:
For the data type miopenInt8, I think the last flag should still be 0. Or this is just a work around for the bug?

Reply:
In fact the entire int8 path is based on int8x4 from the every beginning. This should be OK. Let's see if the CI complains. @scxiao

Reply:
I guess the reason for everything is int8x4 is because rocblas only support int8x4 (root cause is our hardware only supports int8x4). With rocblas supporting int8 (no need to be int8x4), we will have the non-int8x4 input, maybe that can be fixed later.